<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>181</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="MenuFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QGraphicsView" name="ContactImageDisplay">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>161</width>
          <height>221</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="ProfileImage">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>40</y>
          <width>131</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../QtImages/usuario.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AddButton">
       <property name="text">
        <string>Add New Contact</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EditButton">
       <property name="text">
        <string>Edit Contact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteButton">
       <property name="text">
        <string>Delete Contact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ExitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>13</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Contact Image</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="ContactsScrollArea">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>541</width>
      <height>331</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>539</width>
       <height>329</height>
      </rect>
     </property>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>481</width>
        <height>311</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>4</y>
         <width>471</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBoxMenu">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>301</width>
         <height>251</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>New Contact</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QLineEdit" name="PhonelineEdit">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>130</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
        <property name="placeholderText">
         <string>Enter phone's number</string>
        </property>
       </widget>
       <widget class="QLabel" name="GenderLabel">
        <property name="geometry">
         <rect>
          <x>34</x>
          <y>61</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Gender:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="AgespinBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>90</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="AgeLabel">
        <property name="geometry">
         <rect>
          <x>52</x>
          <y>91</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Age:</string>
        </property>
       </widget>
       <widget class="QLabel" name="PhoneLabel">
        <property name="geometry">
         <rect>
          <x>41</x>
          <y>130</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Phone:</string>
        </property>
       </widget>
       <widget class="QLabel" name="NameLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>39</x>
          <y>21</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="NamelineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>100</number>
        </property>
        <property name="placeholderText">
         <string>Enter contact's name</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="CancelButton">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>220</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SaveButton">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>220</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>81</x>
          <y>46</y>
          <width>137</width>
          <height>41</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="GFemaleButton">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Female</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="GMaleButton">
           <property name="text">
            <string>Male</string>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="CurrentContactsLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Current contacts:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ContactsCounterLabel">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>381</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionExit_Alt_F4"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit_Alt_F4">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>../../QtImages/ExitLogo.png</normaloff>../../QtImages/ExitLogo.png</iconset>
   </property>
   <property name="text">
    <string>Exit                    Alt+F4</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../QtImages/QtLogos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddButton</sender>
   <signal>clicked()</signal>
   <receiver>groupBoxMenu</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveButton</sender>
   <signal>clicked()</signal>
   <receiver>groupBoxMenu</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddButton</sender>
   <signal>clicked()</signal>
   <receiver>NamelineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddButton</sender>
   <signal>clicked()</signal>
   <receiver>AgespinBox</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddButton</sender>
   <signal>clicked()</signal>
   <receiver>PhonelineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
